pkg/commands/server/pluginshealth.go

```diff
@@ -44,11 +52,24 @@ var pluginsHealthCommand = cli.Command{
 // cmPluginsHealth is the action for the pluginsHealthCommand. It makes a "plugins" request
 // against the active Synse Server instance and returns plugins' health information.
 func cmdPluginsHealth(c *cli.Context) error {
-	plugins, err := client.Client.Plugins()
+	err := utils.RequiresArgsInRange(0, 3, c)
```

Comment:
not sure why we should limit it to just 3. we could just remove this requirements check entirely, since in theory, there is no limit to the number of tags someone could specify.